cmd/metal-api/internal/grpc/grpc-server.go

```diff
@@ -46,6 +54,7 @@ type ServerConfig struct {
 type Server struct {
 	*WaitService
 	*SupwdService
+	*EventService
```

Comment:
For future we should split up server and service code to make this easier to understand and to decouple for more services to come.